tensorflow_datasets/core/features/features_dict.py

```diff
@@ -171,6 +171,20 @@ def get_serialized_info(self):
         for feature_key, feature in self._feature_dict.items()
     }
 
+  @classmethod
+  def from_json_content(cls, value) -> 'FeatureConnector':
+    features = dict()
+    for feature_key, feature_value in value.items():
+      subclass = cls._REGISTERED_FEATURES.get(feature_value['type'])
```

Comment:
Subclasses should not use the parent internal variable. Don't the following works ?

```py
return cls({
    k: FeatureConnector.from_json(v)
    for k, v in value.items()
})
```